packages/blockchain-wallet-v4-frontend/src/scenes/Login/EnterPassword/index.tsx

```diff
@@ -164,7 +164,7 @@ const EnterPassword = (props: Props) => {
             </Text>
           )}
         </ActionButton>
-        <NeedHelpLink />
+        <NeedHelpLink authActions={props.authActions} origin='PASSWORD' />
```

Comment:
Like here, if you had something like `EnterPasswordOriginEnum.PASSWORD` this would be a little bit cleaner.